packages/astro/src/runtime/server/response.ts

```diff
@@ -3,6 +3,21 @@ const isNodeJS =
 
 let StreamingCompatibleResponse: typeof Response | undefined;
 
+// Undici on Node 16 does not support simply iterating over a ReadableStream
```

Comment:
This is super awkward, on Node 18 there's no issues but it doesn't work with the `undici` package 🤔